### PR TITLE
DONT MERGE: epic template feedback 

### PR DESCRIPTION
**Pull request by p5150j**
### Epic Name:
`[Insert the name of the Epic]`

### Description:
`[Provide a description of the Epic, its business objectives, and its user value.]`

### Features:
`[Identify the features that are necessary to achieve the Epic's business objectives.]`

### User Stories list:
`[Break down the Epic into smaller User Stories that can be completed within a Sprint. Each User Story should be specific, testable, and achievable within the timeframe of a Sprint. Assign User Stories to appropriate teams or team members.]`

### Acceptance Criteria:
`[Define the criteria for determining whether each User Story has been successfully completed. Make sure that the Acceptance Criteria are clear, specific, and verifiable.]`

### Dependencies:
`[Identify any dependencies that may impact the completion of the Epic or its associated User Stories. This may include technical dependencies, resource dependencies, or dependencies on external stakeholders.]`

### Constraints:
`[Identify any constraints that may impact the completion of the Epic or its associated User Stories. This may include budget constraints, time constraints, or other limitations.]`

### Assumptions:
`[Identify any assumptions that have been made about the Epic or its associated User Stories. This may include assumptions about user needs, technical capabilities, or market trends.]`

### Risks:
`[Identify any risks that may impact the completion of the Epic or its associated User Stories. This may include technical risks, resource risks, or risks associated with external factors such as changes in the competitive landscape or regulatory environment.]`

### Ethical Considerations:
`[Identify any ethical considerations associated with the Epic or its associated User Stories. This may include issues related to data privacy, bias, or other ethical concerns. Describe the steps that will be taken to address these concerns.]`

### Design Documents and Reference Artifacts:
`[Provide a list of any design documents or reference artifacts that will be used to guide the development of the Epic. This may include wireframes, mockups, style guides, or other design documents. Describe how these documents will be used by the development team.]`

### Milestones:
`[Define the key milestones that will be used to measure progress towards the completion of the Epic. This may include specific Sprint dates, User Story completion dates, or other milestones that are important to the success of the Epic.]`

### Metrics:
`[Identify the key metrics that will be used to measure the success of the Epic. This may include metrics related to user engagement, customer satisfaction, or revenue generation. Define how these metrics will be tracked and reported.]`

### CODE REVIEW

File: epic.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+Constraints:
+[Identify any constraints that may impact the completion of the Epic or its associated User Stories. This may include budget constraints, time constraints, or other limitations.]
+
+Assumptions:
+[Identify any assumptions that have been made about the Epic or its associated User Stories. This may include assumptions about user needs, technical capabilities, or market trends.]
+
+Risks:
+[Identify any risks that may impact the completion of the Epic or its associated User Stories. This may include technical risks, resource risks, or risks associated with external factors such as changes in the competitive landscape or regulatory environment.]
+
+Ethical Considerations:
```
Comment (mattheworris) [inline epic.md:28]: I like this.

File: epic.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+[Identify any ethical considerations associated with the Epic or its associated User Stories. This may include issues related to data privacy, bias, or other ethical concerns. Describe the steps that will be taken to address these concerns.]
+
+Design Documents and Reference Artifacts:
```
Comment (mattheworris) [inline epic.md:31]: This seems like it would be really helpful.

**Review by mattheworris — COMMENTED** [review]
This is very comprehensive and a great starting point, if there are items that are not applicable, users can delete them.

**Review by aramikm — APPROVED** [review]
I think think is pretty comprehensive and well organized. One other thing to keep in mind is the support part after deployment. What team or people are the owners of maintaining some of these features that are delivered? This might not make sense now since nobody is using our chain but when people started to use it it would be nice to know what team or person should be reached for a certain epic or issue.

**Comment by p5150j** [comment]
> I think think is pretty comprehensive and well organized. One other thing to keep in mind is the support part after deployment. What team or people are the owners of maintaining some of these features that are delivered? This might not make sense now since nobody is using our chain but when people started to use it it would be nice to know what team or person should be reached for a certain epic or issue.

@aramikm I love this idea.. Ill add a support and possibly an IR (incident responds section)

**Review by shannonwells — COMMENTED** [review]
Apparently I never submitted my review from weeks ago

File: epic.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+[Provide a description of the Epic, its business objectives, and its user value.]
+
+Features:
+[Identify the features that are necessary to achieve the Epic's business objectives.]
+
+User Stories list:
+[Break down the Epic into smaller User Stories that can be completed within a Sprint. Each User Story should be specific, testable, and achievable within the timeframe of a Sprint. Assign User Stories to appropriate teams or team members.]
+
+Acceptance Criteria:
+[Define the criteria for determining whether each User Story has been successfully completed. Make sure that the Acceptance Criteria are clear, specific, and verifiable.]
```
[the remark this answers is not in the document]
Comment (p5150j) [reply epic.md:14]: agreed, I do think have a feature Epic AC is good for revisiting the entire scope at the end when possible.. but 💯 agree they need to be in stories in much more detail than at the epic level

File: epic.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+[Identify any ethical considerations associated with the Epic or its associated User Stories. This may include issues related to data privacy, bias, or other ethical concerns. Describe the steps that will be taken to address these concerns.]
+
+Design Documents and Reference Artifacts:
+[Provide a list of any design documents or reference artifacts that will be used to guide the development of the Epic. This may include wireframes, mockups, style guides, or other design documents. Describe how these documents will be used by the development team.]
+
+Milestones:
+[Define the key milestones that will be used to measure progress towards the completion of the Epic. This may include specific Sprint dates, User Story completion dates, or other milestones that are important to the success of the Epic.]
+
+Metrics:
+[Identify the key metrics that will be used to measure the success of the Epic. This may include metrics related to user engagement, customer satisfaction, or revenue generation. Define how these metrics will be tracked and reported.]
```
[the remark this answers is not in the document]
Comment (p5150j) [reply epic.md:38]: 💯 agreed 👍🏻 .. nuking this

File: epic.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+[Identify any constraints that may impact the completion of the Epic or its associated User Stories. This may include budget constraints, time constraints, or other limitations.]
+
+Assumptions:
+[Identify any assumptions that have been made about the Epic or its associated User Stories. This may include assumptions about user needs, technical capabilities, or market trends.]
+
+Risks:
+[Identify any risks that may impact the completion of the Epic or its associated User Stories. This may include technical risks, resource risks, or risks associated with external factors such as changes in the competitive landscape or regulatory environment.]
+
+Ethical Considerations:
+[Identify any ethical considerations associated with the Epic or its associated User Stories. This may include issues related to data privacy, bias, or other ethical concerns. Describe the steps that will be taken to address these concerns.]
```
[the remark this answers is not in the document]
Comment (p5150j) [reply epic.md:29]: yas! Calling it Ethical Impact's and I do like linking directly our ethics paper or something

File: epic.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
+Epic Name:
+[Insert the name of the Epic]
+
+Description:
+[Provide a description of the Epic, its business objectives, and its user value.]
```
[the remark this answers is not in the document]
Comment (p5150j) [reply epic.md:5]: good call, Ill adjust wording ty

**Comment by wilwade** [comment]
Archiving

**Pull request closed without merging**